src/V3Ast.h

```diff
@@ -2566,6 +2576,9 @@ class AstNodeBlock VL_NOT_FINAL : public AstNode {
     AstNode* stmtsp() const { return op1p(); }  // op1 = List of statements
     void addStmtsp(AstNode* nodep) { addNOp1p(nodep); }
     bool unnamed() const { return m_unnamed; }
+    virtual bool isFirstInMyListOfStatements(AstNode* nodep) const override {
+        return (nodep && nodep == stmtsp());
```

Comment:
In all of these, do we need to check nodep for null at all?  It shouldn't be null I'd think, and then there's no harm in the compare.

Also no need for () around the return value.

Reply:
I agree on not needing to check the nodep arg for null, especially given the caller has control over it and our one use case is passing a non-nullptr arg to it. I'm re-running the test regress with that check removed, along with the other suggestions, and if all looks good I'll update the PR tonight/tomorrow.